open-banking-accelerator/components/com.wso2.openbanking.accelerator.common/src/main/java/com/wso2/openbanking/accelerator/common/util/OpenBankingUtils.java

```diff
@@ -50,7 +51,7 @@ public static Object getClassInstanceFromFQN(String classpath) {
             log.error("Class not found: " + classpath.replaceAll("[\r\n]", ""));
             throw new OpenBankingRuntimeException("Cannot find the defined class", e);
         } catch (InstantiationException | InvocationTargetException |
-                NoSuchMethodException | IllegalAccessException e) {
+                 NoSuchMethodException | IllegalAccessException e) {
```

Comment:
```suggestion
                NoSuchMethodException | IllegalAccessException e) {
```

Reply:
Resolved in [696f2d5](https://github.com/wso2/financial-open-banking/pull/62/commits/696f2d5a3f7bccb6c2ef1b3313133ff99bd1eb0d)